### DISCUSSION TITLE
Smoother transitions

### DISCUSSION THREAD

**Discussion by sglushkov** [vchlum/hue-lights#49] [Q&A]
Hi, is there a way or future feature for smoother transition in color/brightness during screen sync ?

**Comment by vchlum** [comment]
Hi @sglushkov! The sync screen has no adjustable transition time like common light use. You can try to increase the intensity in the Entertainment menu.... the more the intensity the smoother should be the overall transitions. Could you try it?

**Comment by sglushkov** [comment]
It is set to maximum intensity. However I can see the transition in fast steps which are not feel like smooth one. I've used the Philips' Hue Sync  app for Windows  before and there these transitions are definitely smooth. Probably adding more steps if possible for intensity and delay for transition will  lead to same smooth feeling. I can check what were the settings in Windows if this can help you in some way. Thank you for your work.
